mmedit/models/editors/gfpgan/stylegan2_clean_modules.py

```diff
@@ -0,0 +1,411 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
I think we'd better hight light the difference between StyleGAN2's original implementation and the clean version in the docstring of each module.